### PR TITLE
feat: add get network intercepts method

### PR DESCRIPTION
**Pull request by thiagowfx**
Uses dependency: https://www.npmjs.com/package/urlpattern-polyfill

Issue: #1183
Spec: https://w3c.github.io/webdriver-bidi/#get-the-network-intercepts
WPT: https://github.com/web-platform-tests/wpt/pull/41662/

### CODE REVIEW

**Comment by thiagowfx** [comment]
Missing:

- `// TODO: Step 4. URL serialization.`
- `static matchUrlPattern`

File: package.json [hunk 1]
```diff
@@ -220,6 +220,7 @@
     "terser": "5.19.2",
     "tslib": "2.6.1",
     "typescript": "5.1.6",
+    "urlpattern-polyfill": "^9.0.0",
```
Comment (Lightning00Blade) [inline package.json:223]: Pin. Also It may be needed in Dependencies, rather then here.

Comment (jrandolf) [inline package.json:223]: Lock the version

File: src/bidiMapper/domains/network/NetworkProcessor.ts [hunk 2]
```diff
@@ -14,6 +14,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import {URLPattern} from 'urlpattern-polyfill';
```
Comment (Lightning00Blade) [inline src/bidiMapper/domains/network/NetworkProcessor.ts:17]: I think we should try to load it only if its not there, like we did with `uuid`.

Comment (thiagowfx) [reply src/bidiMapper/domains/network/NetworkProcessor.ts]: Done

**Pull request merged**